litmus-portal/frontend/src/components/WorkflowCard/styles.ts

```diff
@@ -33,7 +33,7 @@ const useStyles = makeStyles((theme) => ({
   // CardContent
 
   card: {
-    width: theme.spacing(23),
+    width: theme.spacing(26),
```

Comment:
use rem

Reply:
We have been using theme @ishangupta-ds. So is it require?

Reply:
Yes it was wrong we need to change this